### PR TITLE
Add message check

### PR DESCRIPTION
**Pull request by pb82**
This PR includes the timeout for the ditch http call and the endpoint to reply to ditch via message bus. The individual PRs have been reviewed.

### CODE REVIEW

**Comment by pb82** [comment]
retest this please

**Comment by pb82** [comment]
retest this please

**Comment by pb82** [comment]
retest this please

**Comment by pb82** [comment]
retest this please

**Comment by thailekha** [comment]
LGTM

File: lib/messageHandlers/migrationStatusHandler.js [hunk 1]
```diff
@@ -97,6 +97,25 @@ function migrationUpdate(json) {
   });
 }
 
+function onPing(amqpConnection, exchangeName, json, headers, deliveryInfo) {
```
Comment (sedroche) [inline lib/messageHandlers/migrationStatusHandler.js]: Can you add some docs for this function please?

Comment (pb82) [inline lib/messageHandlers/migrationStatusHandler.js]: Ok, will do

**Pull request merged**